circuits/src/transaction.rs

```diff
@@ -184,22 +185,20 @@ impl TxOutputNote {
 ///    deposit, where a deposit refers to funds being transfered to a contract.
 ///
 /// The gadget appends the following public input values to the circuit:
-/// - `skeleton_hash`
 /// - `root`
```

Comment:
Is maybe `payload_hash` missing?

Reply:
it's appended in the TxCircuit,  not the gadget.